contributor-growth/annual-review-email-template.md

```diff
@@ -0,0 +1,36 @@
+# Annual Review TAG CS Email Template
+
+This is a template for an email that we can send out to CNCF projects after they submit their annual review. The goal is to let them know that we are here to help with action items from their review.
+
+The reply-to field for the email should be cncf-tag-contributor-strategy@lists.cncf.io
+___
+
+Hello maintainers!
+
+TAG Contributor Strategy is reaching out to you after your project's annual review to introduce ourselves and offer our assistance with action items from your annual review.
+
+## Who is TAG Contributor Strategy?
+[TAG Contributor Strategy](https://contribute.cncf.io/about/) is a Technical Advisory Group for the CNCF Technical Oversight Committee (TOC). We develop strategies to build, scale, and retain contributor communities and assist the TOC with: project governance, contributor growth, and supporting maintainers. 
+
+## What can we help with?
+We can answer questions and offer guidance on non-technical action items from your annual review. Here are examples of common review feedback that we can help with:
```

Comment:
I kind of hate the word "non-technical" - maybe we can just remove it since we already have examples below?

Reply:
Yeah I was trying to indicate we can't help with everything, just stuff that's relevant to our TAG, but the examples do a good job of highlighting what those would be.